README.md

```diff
@@ -125,7 +133,7 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+# code here Skipping this question. Need assistance.
```

Comment:
My suggestions for how to tackle this question is to start with what you know:
We have two models, `genie` and `lamp`. The relationships looks like this:
`lamp` `has_many` `:genies`,
 `genie` `belongs_to` `:lamp`

Therefore we can start writing code like:

```ruby
lamp = Lamp.create(wishes_remaining: 3)
Genie = Lamp.genie.create(name: "Genie")
lamp.update(wishes_remaining: 1)

Jafar = Genie.create(name: "Jafar")
Jafar.lamp.create(wishes_remaining: 3)

Genie.update(lamp_id: nil)
```